src/mugs.js

```diff
@@ -444,176 +601,6 @@ define([
-                }
-                return [key, val];
-            }));
-        }
-
-        this.ufid = util.get_guid();
-        this.form = form;
-        this._baseSpec = baseSpec;
-        this.setOptionsAndProperties(options, properties);
-    }
-    Mug.prototype = {
-        // set or change question type
-        setOptionsAndProperties: function (options, properties) {
-            var _this = this,
-                currentAttrs = properties || (this.p && this.p.getAttrs()) || {};
-
-            // These could both be calculated once for each type instead of
-            // each instance.
-            this.options = util.extend(defaultOptions, options);
-            this.__className = this.options.__className;
-            this.spec = copyAndProcessSpec(this._baseSpec, this.options.spec, this.options);
-
-            // Reset any properties that are part of the question type
-            // definition.
-            _.each(this.spec, function (spec, name) {
-                if (spec.deleteOnCopy) {
-                    delete currentAttrs[name];
-                }
-            });
-
-            this.p = new MugProperties({
-                spec: this.spec,
-                mug: this,
-                shouldChange: _this.form.shouldMugPropertyChange.bind(_this.form),
-            });
-            this.options.init(this, this.form);
-            this.p.setAttrs(currentAttrs);
-        },
-        getAppearanceAttribute: function () {
-            return this.options.getAppearanceAttribute(this);
-        },
-        getIcon: function () {
-            return this.options.getIcon(this);
-        },
-        getErrors: function () {
-            return this.p.getErrors();
-        },
-        isValid: function () {
-            return !this.getErrors().length;
-        },
-        /*
-         * Gets a default label, auto-generating if necessary
-         */
-        getDefaultLabelValue: function () {
-            var label = this.p.label,
-                nodeID = this.p.nodeID;
-            if (label) {
-                return label;
-            } else if (nodeID) {
-                return nodeID;
-            } else if (this.__className === "Item") {
-                return this.p.defaultValue;
-            }
-        },
-        
-        /*
-         * Gets the actual label, either from the control element or an empty
-         * string if not found.
-         */
-        getLabelValue: function () {
-            var label = this.p.label;
-            if (label) {
-                return label;
-            } else {
-                return "";
-            } 
-        },
-        
-        // Add some useful functions for dealing with itext.
-        setItextID: function (val) {
-            var labelItext = this.p.labelItext;
-            if (labelItext) {
-                labelItext.id = val;
-            }
-        },
-        
-        getItext: function () {
-            return this.p.labelItext;
-        },
```

Comment:
The above two (unused) Itext functions were removed during the code move. Should have at least done that as a separate commit...